### ISSUE TITLE
Allow "constant-style" VariableNaming in build.gradle.kts

### ISSUE THREAD

**Issue by elliot-nelson** [detekt/detekt#6230]
## Expected Behavior of the rule

The `VariableNaming` rule should be relaxed and/or suppressed in `build.gradle.kts` files.

## Context

I find that we have many examples of "constants" in `build.gradle.kts` files, usually defined up top somewhere, like:

```kotlin
val TEAM_ID = 'some team name';
val APP_ID = 'some id';
// etc...
```

Because these are used as constants, renaming them to lowerCamelCase makes their usage really confusing to read. But adding `@Suppress` to dozens of files isn't great either. And unfortunately you can't use `const val` inside a script file...

(If there's some other way to get what I want -- basically allow "constant-style" names for variables in kotlin script files, without having to suppress or baseline them, I'm open to ideas!)

**Comment by cortinico** [comment]
This will practically be fixed by:
https://github.com/detekt/detekt/discussions/6035#discussioncomment-5702491

**Comment by BraisGabin** [comment]
Right now you could exclude all the `*.kts` from that rule. And this will be even more flexible on 2.0 when the feature that Nico links is implemented.

For those reasons I'm closing this issue. If I'm missing something please don't hesitate to comment.
